src/webapp/pages/NeighborhoodFinder.js

```diff
@@ -95,91 +101,102 @@ export default function NeighborhoodFinder() {
 
   return (
     <Box className={classes.root}>
-      <Typography className={classes.text} variant="body1">
-        Enter an address and we will look up cross streets and the neighborhood.
-      </Typography>
-      <Typography className={classes.text} variant="body1">
-        For best results, enter street and town (Ex: 1550 dean st brooklyn)
-      </Typography>
-      <Typography className={classes.text} variant="body1">
-        The address will not be stored or logged :)
-      </Typography>
-      <form onSubmit={handleSubmit} autoComplete="off">
-        <TextField
-          id="address"
-          name="address"
-          label="Address"
-          type="text"
-          margin="normal"
-          variant="outlined"
-          required
-          onChange={e => setAddress(e.target.value)}
-          className={classes.field}
-          InputProps={{
-            endAdornment: (
-              <InputAdornment position="end">
-                <Button aria-label="address-submit" onClick={handleSubmit}>
-                  Submit
-                </Button>
-              </InputAdornment>
-            )
-          }}
-        />
-      </form>
-
-      {data && (
-        <>
-          <TextField
-            disabled
-            id="cross-1"
-            label="Cross Street #1"
-            value={data.intersection.street_1}
-            variant="outlined"
-            className={classes.field}
-          />
-          <TextField
-            disabled
-            id="cross-2"
-            label="Cross Street #2"
-            value={data.intersection.street_2}
-            variant="outlined"
-            className={classes.field}
-          />
-          <TextField
-            disabled
-            id="neighborhood"
-            label="Neighborhood"
-            value={data.neighborhoodName || "Unavailable"}
-            helperText="If both this and zone are unavailable, double check the map: https://bit.ly/2UrZPkA"
-            variant="outlined"
-            className={classes.field}
-          />
+      <Box className={classes.formRoot}>
+        <Box>
+          <Typography className={classes.text} variant="h3">
```

Comment:
add heading for readability

Reply:
do you mind explaining why aim for minimal text? from a UI perspective, if users have a number of tabs open, it will be easier for them to pick out if they're on the right one if there's an obvious title. it's also helpful for those who are new to the tool.

Reply:
> if users have a number of tabs open, it will be easier for them to pick out if they're on the right one if there's an obvious title

I think this is more on the favicon, which is still the CRA one :/

Thought I think you're right. Can we just make it much smaller? h5? No need to update the screenshots imo

Reply:
yeah agree re: the favicon. that would be a different task. i was thinking more if you're cycling through tabs. tbh if we're doing it the right way it should still be a h1 with a smaller font size but probably not worth twiddling that right now. pushed update!